### PR TITLE
Notifications List component

### PR DESCRIPTION
**Pull request by marcinbodnar**
It's not the final version of the component, but it's the proper layout for further work.

### CODE REVIEW

File: src/NearOrg/Notifications/NotificationsBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+const bellIcon = (
```
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/NotificationsBanner.jsx:1]: Let's remove this icon from here and this use https://phosphoricons.com instead

File: src/NearOrg/Notifications/NotificationsBanner.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  font: var(--text-base);
+  font-weight: 600;
+`;
+
+const Buttons = styled.div``;
+
+return (
+  <Card>
+    <Component>
+      <Icon>{bellIcon}</Icon>
```
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/NotificationsBanner.jsx:68]:
```suggestion
      <Icon>
        <i class="ph ph-bell-ringing" />
      </Icon>
```

File: src/NearOrg/Notifications/NotificationsBanner.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          props={{
+            href: "",
+            label: "No thanks",
+            variant: "secondary",
+            size: "default",
+          }}
+        />
+      </Buttons>
+    </Component>
+    <Close>{closeIcon}</Close>
```
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/NotificationsBanner.jsx:91]:
```suggestion
    <Close>
      <i class="ph ph-x" />
    </Close>
```

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

**Comment by charleslavon** [comment]
@marcinbodnar should this PR be closed in favor of #330?

**Pull request closed without merging**